src/Nethermind/Nethermind.Core.Test/Blockchain/TestBlockchain.cs

```diff
@@ -363,11 +363,6 @@ protected virtual Block GetGenesisBlock()
             genesisBlockBuilder.WithParentBeaconBlockRoot(Keccak.Zero);
         }
 
-        if (SpecProvider.GenesisSpec.RequestsEnabled)
-        {
-            genesisBlockBuilder.WithConsensusRequests(0);
```

Comment:
I think it should be likely replaced with blockHeader.RequestsHash = EmptyRequestsRoot

Reply:
Gotcha!